pkgs/top-level/all-packages.nix

```
@@ -4443,6 +4443,8 @@ with pkgs;
     autoreconfHook = buildPackages.autoreconfHook269;
   };
 
+  emote = callPackage ../tools/inputmethods/emote { };
```

```suggestion
  emote = with python3Packages; toPythonApplication emote;
```

and add `emote = callPackage ../tools/inputmethods/emote { };` to pkgs/top-level/python-packages.nix